modules/sdk/project-templates-theme/src/main/resources/archetype-resources/pom.xml

```diff
@@ -54,7 +54,7 @@
 				</executions>
 				<configuration>
 					<docrootDirName>${com.liferay.portal.tools.theme.builder.outputDir}</docrootDirName>
-					<precision>5</precision>
+					<precision>9</precision>
```

Comment:
I changed the precision to 9 because these new templates use the latest version of `gradle-plugins-css-builder`, which has 9 as default.